### DISCUSSION TITLE
[Feature request] Support the build in navigate back.

### DISCUSSION THREAD

**Discussion by generic-user** [vslinko/obsidian-zoom#54] [Ideas]
Currently, when we click on the navigate back button (Go Back), the zoom is not undone. This would be very useful.
![image](https://user-images.githubusercontent.com/1516035/152160422-4426c6a4-1776-4a7e-8bb9-384d73e79cbf.png)

**Comment by vslinko** [comment]
ref https://discord.com/channels/686053708261228577/840286264964022302/938519860571406346

**Reply by samlevan** [reply]
Ditto on this feature request. I am of that group of users who uses one pane mostly and use the keyboard to quickly navigate back and forth between notes (either "go back" and "go forward", or opening a note with the quick switch). Not remembering the Zoom state on a note makes the plugin almost unusable for this kind of workflow. I personally don't see a downside to always remembering the Zoom state a note was at. Thanks for considering this point of view, this plugin has so much potential when combined with the outliner plugin.
